.gitignore

```diff
@@ -12,7 +12,7 @@ npm-debug.log
 reports/*.xml
 veracode
 artifacts
-
+*.tsbuildinfo

```

Comment:
this is generated by composite project and can be used for incremental rebuild (but we ain't there yet, hold ye 🐴 )